src/SkirtBrim.cpp

```diff
@@ -80,8 +81,8 @@ void SkirtBrim::getFirstLayerOutline(SliceDataStorage& storage, const size_t pri
     first_layer_outline = first_layer_outline.offset(join_distance).offset(-join_distance); // merge adjacent models into single polygon
     constexpr coord_t smallest_line_length = 200;
     constexpr coord_t largest_error_of_removed_point = 50;
-    first_layer_outline.simplify(smallest_line_length, largest_error_of_removed_point); // simplify for faster processing of the brim lines
-    if (first_layer_outline.size() == 0)
+    first_layer_outline = Simplify(smallest_line_length, largest_error_of_removed_point, 0).polygon(first_layer_outline);
```

Comment:
Are you sure these arguments weren't meant as squares originally? I know it _says_ length, not len2 or something, but they seem relatively big.

Reply:
No, this was part of the confusion of the original function, really.

`PolygonRef::simplify` indeed takes squared distances.
`Polygons::simplify` does NOT take squared distances, but squares the value itself before passing it to `PolygonRef::simplify` for each of its sub-polygons.

This confusion is why I opted for simply not accepting squared distances at all any more. Keeping it simple!